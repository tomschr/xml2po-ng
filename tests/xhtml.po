msgid ""
msgstr ""
"Project-Id-Version: xhtml test\n"
"POT-Creation-Date: 2006-12-26 20:18+0100\n"
"PO-Revision-Date: 2006-12-26 20:19+0100\n"
"Last-Translator: Данило Шеган\n"
"Language-Team: Српски\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: xhtml.xml:5(title)
msgid "Sample XHTML page"
msgstr "Пример XHTML стране"

#: xhtml.xml:7(p)
msgid "This is just an example."
msgstr "Ово је само пример."

#: xhtml.xml:10(img:title)
msgid "Lovely image"
msgstr "Дивна слика"

#: xhtml.xml:10(img:alt)
msgid "Image of nothing"
msgstr "Слика ничега"

#. Here nothing but a paragraph should be extracted.
#: xhtml.xml:13(p)
msgid "Enough for a test, if you ask <a href=\"mailto:danilo\" title=\"Danilo\">me personally</a>."
msgstr "Довољно за пробу, ако питате <a href=\"mailto:danilo\" title=\"Данила\">мене лично</a>."
