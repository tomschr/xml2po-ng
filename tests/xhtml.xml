<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head><title>Sample XHTML page</title></head>
<body>
<p>This is just an example.</p>

<!-- Here both title and alt attributes should be extracted. -->
<img title="Lovely image" alt="Image of nothing" src="nothing.png" />

<!-- Here nothing but a paragraph should be extracted. -->
<p>Enough for a test, if you ask <a href="mailto:danilo" title="Danilo">me personally</a>.</p>
</body>
</html>
