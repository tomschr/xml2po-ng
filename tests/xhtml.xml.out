<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><title>Пример XHTML стране</title></head>
<body>
<p>Ово је само пример.</p>

<!-- Here both title and alt attributes should be extracted. -->
<img title="Дивна слика" alt="Слика ничега" src="nothing.png" />

<!-- Here nothing but a paragraph should be extracted. -->
<p>Довољно за пробу, ако питате <a href="mailto:danilo" title="Данила">мене лично</a>.</p>
</body>
</html>
